friends/src/data.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
```

Comment:
I would not name this file data. That makes things confusing. Data file usually holds data like an array of object. Also would want to have this inside of your components.